src/types/model/customType.ts

```diff
@@ -1,54 +1,54 @@
 import { CustomTypeModelField } from "./types";
 
 /**
- * A Prismic Custom Type model.
+ * A Prismic custom type model.
  *
- * @typeParam ID - API ID of the Custom Type.
- * @typeParam Definition - The Custom Type's tabs and their fields.
+ * @typeParam ID - API ID of the custom type.
+ * @typeParam Definition - The custom type's tabs and their fields.
  */
 export interface CustomTypeModel<
 	ID extends string = string,
 	Definition extends CustomTypeModelDefinition = CustomTypeModelDefinition,
 > {
 	/**
-	 * The ID of the Custom Type model.
+	 * The ID of the custom type model.
 	 */
 	id: ID;
 
 	/**
-	 * The human readable name of the Custom Type Model.
+	 * The human readable name of the custom type Model.
```

Comment:
Might want to lowercase "Model" for consistency.

Reply:
Probably missed this one using find and replace. I checked the rest of the repo for "custom type Model" and confirmed there are no other instances of this. 👍